releases/release-6.1.1.md

```diff
@@ -0,0 +1,121 @@
+---
+title: TiDB 6.1.1 Release Notes
+---
+
+# TiDB 6.1.1 Release Notes
+
+Release date: 2022-xx-xx
+
+TiDB version: 6.1.1
+
+## Compatibility changes
+
++ TiDB
+
+    (dup: release-6.2.0.md > Bug fixes> TiDB)- Fix the issue that `SHOW DATABASES LIKE …` is case-sensitive [#34766](https://github.com/pingcap/tidb/issues/34766)
+
+## Improvements
+
++ TiDB
+
+<!-- <planner> -->
+(dup: release-6.2.0.md > # Performance)[User document](/optimizer-hints.md#semi_join_rewrite) [#35323](https://github.com/pingcap/tidb/issues/35323)
+(dup: release-5.2.4.md > Bug fixes> TiDB)- Fix the issue that partitioned tables cannot fully use indexes to scan data in some cases [#33966](https://github.com/pingcap/tidb/issues/33966)
+
+<!-- <transaction> -->
+(dup: release-6.2.0.md > Bug fixes> TiDB)- Avoid sending requests to unhealthy TiKV nodes to improve availability [#34906](https://github.com/pingcap/tidb/issues/34906)
+
++ TiKV
+
+    - In the new backup organization structure, we will see: `./br backup --pd "127.0.0.1:2379" -s "s3://backup/20220621" - After br command finished, we will have the structure below. ➜ backup tree . . └── 20220621 ├── backupmeta ├── store1 │ └── backup-xxx.sst ├── store100 │ └── backup-yyy.sst ├── store2 │ └── backup-zzz.sst ├── store3 ├── store4 └── store5` [#13063](https://github.com/tikv/tikv/issues/13063)
+    (dup: release-6.2.0.md > Improvements> TiKV)- Support compressing the metrics response using gzip to reduce the HTTP body size [#12355](https://github.com/tikv/tikv/issues/12355)
+
+## Bug fixes
+
++ TiDB
+
+<!-- <execution> -->
+- executor: fix index_lookup_hash_join hang when used with limit [#35638](https://github.com/pingcap/tidb/issues/35638)
+
+<!-- <planner> -->
+- planner: fix outer join reorder will push down its outer join condition [#37238](https://github.com/pingcap/tidb/issues/37238)
+- planner: fix cte-schema-clone will clone the old hashcode of its column if any [#35404](https://github.com/pingcap/tidb/issues/35404)
+
+<!-- <sql-infra> -->
+(dup: release-6.2.0.md > Bug fixes> TiDB)- Fix the issue that querying partitioned tables might report "index-out-of-range" and "non used index" errors in some cases [#35181](https://github.com/pingcap/tidb/issues/35181)
```

Comment:
- Fix the issue that querying partitioned tables might report "index-out-of-range" and "non-used index" errors in some cases [#35181](https://github.com/pingcap/tidb/issues/35181)
- Fix the issue that when using TiDB with Binlog, the Drainer may crash because the invalid schema version after `ALTER SEQUENCE` statement [#36276](https://github.com/pingcap/tidb/issues/36276)
- Fix the incorrect TiDB states that may appear on startup under very extreme cases [#36791](https://github.com/pingcap/tidb/issues/36791)
- Fix the issue that the execution plans for the partition table may show `UnknownPlanID` in TiDB Dashboard. [#35153](https://github.com/pingcap/tidb/issues/35153)